### DISCUSSION TITLE
Keybase vs. Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#1080] [Comments]
# Keybase vs. Ubuntu 24.04

Trying to install Keybase "the normal way" on Ubuntu leads to errors about missing packages. Trying then to run `apt install -f` to fix the issues will lead to Keybase b…

https://kollitsch.dev/blog/2024/installing-keybase-on-ubuntu-2404/

**Comment by milegrin** [comment]
I got around this dependency for FortiClient by temporarily adding the Lunar (Or Jammy - your choice) "main" and "universe" repo's; installing the required file sets then removing the lunar/jammy repo before doing anything else.
1) Add temp repo
2) sudo apt update (NOT upgrade!)
3) sudo apt install libappindicator1 libgconf-2-4  (libayatana-appindicator1 conflicts with libappindicator1)
4) Remove temp repo
5) sudo apt update
6) App should no install without issues

Your results may vary :)

**Comment by davidsneighbour** [comment]
I found out some weeks ago that the latest version they published now installs without issues on Ubuntu 24.04. This solves the installation issues. The issues that "some entitity has taken over" the program and is not developing it any further and the program probably being insecure by abandonment now or soon won't be solved. I am removing all uses of Keybase. It's one of these "no hope" cases (Zoom bought them some years ago and the original developer team is not part of it anymore).
